# This shows what happens when a function fails to keep its oath.

def main():
    innocent_fn()
